auditlog/models/http_request.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from openerp.http import request
+
+
+class AuditlogHTTPRequest(models.Model):
+    _name = 'auditlog.http.request'
+    _description = u"Auditlog - HTTP request log"
+    _order = "create_date DESC"
+    _rec_name = 'display_name'
+
+    display_name = fields.Char(u"Name", compute="_display_name")
```

Comment:
any reason not to overwrite `name_get` instead? This will be the display name by default, with the added benefit that `name_get` also returns something sensible. Then you also don't need to set `_rec_name`. cf https://github.com/OCA/OCB/blob/9.0/openerp/models.py#L1664